bin/circleci/install-test-dependencies.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 set -ex
 
+IS_PONTOON=$(git show -s --format=%s | grep -q 'Pontoon:' && echo 'true' || echo '')
+if [[ $IS_PONTOON ]]; then
+  echo "Skipping Storybook deploy on Pontoon commit.";
```

Comment:
I think this was a copy paste error, it should say Skipping Integration Tests instead?